checkout/etc/frontend/di.xml

```diff
@@ -21,4 +21,16 @@
             </argument>
         </arguments>
     </type>
+
+    <!-- LiqPay Payment Configuration -->
+    <virtualType name="PronkoLiqPayPaymentConfig" type="Magento\Payment\Gateway\Config\Config">
```

Comment:
This virtualType is not needed as we have similar one Pronko\LiqPayGateway\Gateway\Config\Config

Reply:
@mcspronko Done.